_posts/2022-10-13-opening-reception.md

```diff
@@ -0,0 +1,30 @@
+---
+author: Lacey Williams Henschel
```

Comment:
```suggestion
author: DjangoCon US Organizers
```

Reply:
I am not against giving Lacey credit but we shouldn't publish this without getting her approval first